run watir specs on Sauce Labs

titusfortner opened the pull request:
@p0deje - I'm looking for feedback on how I implemented this in Watirspec. This is working but I'm not sure I'm approaching it the right way.

The heavy lifting for Sauce integration is done with the Saucer gem that is still very much in a beta mode. (I need to see if knapsack works better because I'm struggling with how parallel split test is doing things right now).

titusfortner committed:
run watir specs on Sauce Labs

p0deje commented:
1. Why do you need to change hooks from `before(:all)` to `before(:each)`?
2. Why do you need to run specs in parallel? Isn't 1 process enough for us?

titusfortner commented:
I'm not sure; for some reason that spec wasn't working with this code and I changed it to `before(:each)` and now it is working. I didn't figure out the cause, just changed the code. I could look into it more.

I'm not looking to replace what we have now. Partly this is an exercise to make sure we can run specs on Sauce and get a working example with the saucer gem; though, it would also be nice if it were simple to run tests on any platform/browser.

titusfortner commented:
This is now out of date and needs reworking, will resubmit later.
